naturewatch_camera_server/static/client/src/settings/TimelapseSetting.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import {ToggleButtonGroup, ToggleButton} from 'react-bootstrap';
+
+class TimelapseSetting extends React.Component {
+
+    onIntervalChange(event) {
+        this.props.onIntervalChange(event.target.value);
+    }
+
+    onIntervalChangeEnd(event) {
+        this.props.onIntervalEnd(event.target.value);
+    }
+
+    renderIntervalText(seconds) {
+        if (seconds > 59) {
+            return Math.round((seconds / 60)) + " minutes";
```

Comment:
We should probably either display the number of seconds as well (the easy way), or "snap" into minutes. Ideally, the slider would be a logarithm (beginning is 10s by 10s, then minutes, then 5 minutes, then 10).

Reply:
Can you suggest a minimum / maximum time for the timelapse photography? In terms of minimum, I'd suggest 10 seconds to avoid any performance issues with taking photos too quickly.

Reply:
10s is just perfect as a minimum. More than one hour does not really make sense for me. Maybe max 2h?

Reply:
What I mean is:
- from 10s to 1:30min, you go 10s by 10s.
- then minutes (1 by 1) until 10, where it switches to 5min by 5min.
- from 1 hour on, it's 10min by 10min.